### PR TITLE
Fix swap gas limit estimation

### PR DESCRIPTION
**Pull request by FrederikBolding**

### CODE REVIEW

File: src/services/ApiService/Dex/Dex.ts [hunk 1]
```diff
@@ -118,6 +118,7 @@ export default class DexService {
       data.allowanceTarget !== AddressZero
         ? {
             ...formatApproveTx({
+              fromAddress: (account as TAddress) ?? undefined,
```
Comment (Mrtenz) [inline src/services/ApiService/Dex/Dex.ts:121]: `?? undefined` seems redundant here?

**Review by Mrtenz — CHANGES_REQUESTED** [review]

File: src/services/ApiService/Dex/Dex.ts (hunk 1 above)
Comment (FrederikBolding) [reply src/services/ApiService/Dex/Dex.ts:121]: No, because `null` and `undefined` aren't the same

Comment (FrederikBolding) [reply src/services/ApiService/Dex/Dex.ts:121]: `account` can be `string` or `null`

Comment (Mrtenz) [reply src/services/ApiService/Dex/Dex.ts:121]: Can't you change the type of account to `string | undefined` then?

Comment (FrederikBolding) [reply src/services/ApiService/Dex/Dex.ts:121]: Sure

**Review by Mrtenz — APPROVED** [review]

**Pull request merged**